docs/2_4_common_schema.adoc

```diff
@@ -1710,9 +1710,11 @@ Knowns latexmath:[{\mathbf{v}_{\mathit{known}}}] in <<InitializationMode>> (for
 
 * <<independent>> variable (usually time; <<causality>> = <<independent>>).
 
-_[For a dynamic system most of the unknowns would have an implict dependency on the independent variable.
-When the independent variable is listed among the dependencies this implicit dependency is not what is meant, but rather that the unknown's partial derivative with respect to the independent variable can be non-zero.
-For a mechanical system this is called a rheonomous system, while if the independent variable is not listed in the dependencies, it is a scleronomous system.]_
+_[The <<independent>> variable should be treated as any other variable when it comes to dependencies._
+_As only explicit dependencies are listed it is required that unknowns that depends on the <<independent>> variable have a partial derivative that is different from zero._
```

Comment:
depends --> depend

"... have a partial derivative that is different from zero." 
"... have a partial derivative w.r.t this variable that is different from zero." 

"... have a partial derivative that is different from zero." : this is not true for all times, but the variable _can_ be nonzero.